### ISSUE TITLE
variableMeasured: should SOSO recommend specific property ontologies?

### ISSUE THREAD

**Issue by smrgeoinfo** [ESIPFed/science-on-schema.org#141]
A key to the utility of the schema:PropertyValue interoperability is consistent specification of what the the property is about, as specified by schema:propertyID.    Does SOSO want to recommend a specific vocabulary for better interoperability?

# Some Property vocabularies
Property URIs that might be used in the so:propertyID field. 

  - **CF names** [CF standard name table]( https://cfconventions.org/Data/cf-standard-names/73/src/cf-standard-name-table.xml); names have an id, but no namespace appears to be defined, so there aren't dereferenceable URIs it seems.
  - **SWEET** has a set of property labels with some hierarchical structure providing weak semantics, but noe explicit definitions in the http://sweetontology.net/prop/Property namespace. Properties are at the conceptual level, e.g. 'total alkalinity', 'pH', 'precision', 'temperature range'. 
  - **LTER Measurements**   Provides [A word net, similar to SWEET, but doesn't provide URI or definition](https://vocab.lternet.edu/vocab/vocab/index.php?tema=667&/measurements).  All term URIs are query fragments on the 'vocab/index.php' resource.
  - **Structured Variable ontology** (SVO) SVO provides a [vocabulary of measured variables based on its model](http://geoscienceontology.org/svo/svl/variable/1.0.0/), it is apparently only accessible on a web page with URIs that are html fragment identifiers.
- Wikidata 
- [QUDT quanityKinds]( http://www.qudt.org/doc/DOC_VOCAB-QUANTITY-KINDS.html)
- [Minimum Information about any (x) Sequence (MIxS)](https://gensc.org/mixs/) the GSC family of minimum information standards – 
- [USGS NWIS parameters](https://help.waterdata.usgs.gov/codes-and-parameters/parameters). 
- [Scientific Variables Ontology](http://www.geoscienceontology.org/svo/svl/property/1.0.0/)
- [US EPA substance registry](https://ofmpub.epa.gov/sor_internet/registry/substreg/LandingPage.do)

**Comment by mbjones** [comment]
We also use measurement type classes for variable annotations from:

 - [ENVO](http://www.obofoundry.org/ontology/envo.html)
 - [ECSO](https://bioportal.bioontology.org/ontologies/ECSO)
 - Core observation concepts from [OBOE](https://bioportal.bioontology.org/ontologies/OBOE) and [SSN/SOSA](https://www.w3.org/TR/vocab-ssn/)
 - And many [OBO Foundry](http://www.obofoundry.org) vocabularies

**Comment by smrgeoinfo** [comment]
links to measurement types in ENVO and ECSO
- ENVO measurement types, subclasses of [quality](https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0000001&viewMode=All&siblings=false)?
- The Ecosystem Ontology (ECSO) [Measurement Type](https://bioportal.bioontology.org/ontologies/ECSO/?p=classes&conceptid=http%3A%2F%2Fecoinformatics.org%2Foboe%2Foboe.1.2%2Foboe-core.owl%23MeasurementType)

**Comment by rduerr** [comment]
I think any legitimate publicly accepted and accessible semantic resource should be fair game - that would include W3C-type ontologies, OBOFoundry ontologies, SWEET,  and the other resources mentioned above.  Since there are different communities involved they need to be able to specify their own terms!

Suggest closing with a decision of NO

**Comment by smrgeoinfo** [comment]
The challenge is that if there is not a recommended vocabulary to use, interoperability becomes problematic.

**Comment by rduerr** [comment]
This is true... but interoperability is already more than problematic.  I don't have a problem with particular communities who want to play nicely together to jointly agree on some term set.  The issue becomes harder when dealing with interoperability across communities where only a few fundamental concepts may be held in common.  Classic example I'm dealing with at the moment.  Biology community insisting that species name is a relevant discovery parameter; which doesn't sit well with the Astrophysicists in the community who insist on other things like Stellar Classification (O, B, A, F, G, K, M).  Both could probably agree on an object type of some sort (astronomical entity, biological organism) but not much below that.

**Comment by smrgeoinfo** [comment]
Good point.  I guess its a question the scope of users you want to accommodate. For a cross domain situation (e.g. trying to work with biologists and astronomers ...) you'd probably need to think of that property as EntityType (i.e. what is this data record about).   Its [data type would be categorical](https://github.com/ESIPFed/science-on-schema.org/blob/issue27-measuredVariable/guides/Issue27-UpdatesForDatasetVariables.md#value-range-is-controlled-vocabulary), and the biologist could specify the range to be a vocabulary of taxa, and the astronomer would specify a vocabulary of star types.    One suggestion that has been made is that the property ID could have multiple values, to account for different levels of granularity in the DDI variable cascade.  (e.g. EntityType, stellar class, stellarClassCode from a particular codebook)
On the other hand, if the metadata is only scoped to a single community, then this isn't necessary.

**Comment by dr-shorthair** [comment]
I generally agree that it is not possible, or necessarily desirable, to limit the set of permitted observed-property/variable-measured values to one or even a small number of vocabularies. 

While it is useful to model good practice, the market will decide. While there is some activity  to get some community agreement on these things (e.g. I-ADOPT in RDA),  there will always be legitimate technical variations, particularly between disciplines, and also innovations.

**Comment by smrgeoinfo** [comment]
Maybe the operational question here is 'what is the cardinality of propertyID'.  Allowing multiple values would allow for at least some accommodation for' legitimate technical variations, particularly between disciplines, and also innovations.'.
